src/commands/languages/index.ts

```diff
@@ -2,25 +2,30 @@ import { colorPalette, commands } from '../../constants'
 import { CommandError, handleError, konsola } from '../../utils'
 import { getProgram } from '../../program'
 import { session } from '../../session'
-import { pullLanguages, saveLanguagesToFile } from './actions'
+import { fetchLanguages, saveLanguagesToFile } from './actions'
 import chalk from 'chalk'
 import type { PullLanguagesOptions } from './constants'
 
 const program = getProgram() // Get the shared singleton instance
 
-export const pullLanguagesCommand = program
-  .command('pull-languages')
-  .description(`Download your space's languages schema as json`)
+export const languagesCommand = program
+  .command('languages')
```

Comment:
@alvarosabu can I suggest to add an alias() `lang` here? I'm a fan of shortcuts 😄

Reply:
Done @edodusi 😉, you can't say I don't spoil you haha